Lesson23/Kubra/filter.js

```diff
@@ -352,6 +358,9 @@ const studentsExercise9 = [
 		grade: 'B-',
 	},
 ];
+const studentsExercise9Result = studentsExercise9.filter(student => student.grade[0] !== "D" && student.grade !== "F");
```

Comment:
This should actually be || (OR) instead of AND as the student only has one grade at a time